simmechanics/data/icub3/ICUB_3_all_options.yaml

```diff
@@ -335,6 +335,29 @@ sensors:
     sensorType: "depth"
     sensorBlobs:
     - |
+        <camera name="intel_realsense_depth_camera">
+          <horizontal_fov>1.02259</horizontal_fov>
+          <distortion>
+            <k1>0</k1>
+            <k2>0</k2>
+            <k3>0</k3>
+            <p1>0</p1>
+            <p2>0</p2>
+            <center>319.5 239.5</center>

```

Comment:
Where does this parameters come from?

Reply:
I copied them from the asus xtion because they were already present in the cer-sim sdf (see https://github.com/robotology/cer-sim/blob/master/gazebo/cer/cer.sdf) for having something to test, but now I have to take them from the datasheet!
I put WIP for this and other things I need to investigate